.gitignore

```diff
@@ -11,3 +11,4 @@ yarn-error.log
 /fixtures/my-remix-app
 
 .eslintcache
+/.idea/
```

Comment:
This should be in your global `.gitignore` file

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

```suggestion
```

Reply:
or if you don't want it global, in .git/info/exclude